src/ontology/uberon-edit.obo

```diff
@@ -224189,6 +224189,69 @@ intersection_of: part_of UBERON:0005610 ! iliac vein
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-06-17T14:34:50Z xsd:dateTime
 
+[Term]
+id: UBERON:8480001
+name: capillary of brain
+def: "A capillary that is part of the brain." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001982 ! capillary
+intersection_of: part_of UBERON:0000955 ! brain
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-06-24T09:28:31Z xsd:dateTime
+
+[Term]
+id: UBERON:8480002
+name: thoracic aorta smooth muscle tissue
+def: "A portion of smooth muscle tissue that is part of the thoracic aorta." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001135 ! smooth muscle tissue
+intersection_of: part_of UBERON:0001515 ! thoracic aorta
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-06-24T09:30:54Z xsd:dateTime
+
+[Term]
+id: UBERON:8480003
+name: iliac artery smooth muscle tissue
+def: "A portion of smooth muscle tissue that is part of an iliac artery." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001135 ! smooth muscle tissue
+intersection_of: part_of UBERON:0005609 ! iliac artery
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-06-24T09:32:07Z xsd:dateTime
+
+[Term]
+id: UBERON:8480004
+name: iliac vein endothelium
+def: "An endothelium that is part of an iliac vein." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001986 ! endothelium
+intersection_of: part_of UBERON:0005610 ! iliac vein
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-06-24T09:33:18Z xsd:dateTime
+
+[Term]
+id: UBERON:8480005
+name: placental artery endothelium
+def: "An endothelium that is part of the placental artery." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001986 ! endothelium
+intersection_of: part_of UBERON:8480007 ! placental artery
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-06-24T09:34:25Z xsd:dateTime
+
+[Term]
+id: UBERON:8480006
+name: mesenteric lymphatic vessel
+def: "Lymphatic vessel that is part of the mesentery." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001473 ! lymphatic vessel
+intersection_of: part_of UBERON:0002095 ! mesentery
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-06-24T09:35:29Z xsd:dateTime
+
+[Term]
+id: UBERON:8480007
+name: placental artery
```

Comment:
sorry, would you mind adding placental vein too? I think if I rmbr right, the modeling would make it so that this is will be the only child of placenta blood vessel which is not great (see https://douroucouli.wordpress.com/2022/01/04/ontotip-avoid-the-single-child-anti-pattern/)

Reply:
OK, I will dot this right now